frontend/src/components/MapView/LeftPanel/ChartsPanel/index.tsx

```diff
@@ -197,7 +197,12 @@ export const downloadCsv = (
         const first = value[0];
         const keys = Object.keys(first);
         const filtered = keys.filter(x => x !== dateColumn);
-        const mapped = filtered.map(x => getKeyName(x, key));
+        // created the column names and filters the anomaly_avg column names from the columns collection
+        const mapped = filtered
+          .map(x => getKeyName(x, key))
+          .filter(keyName => {
+            return !keyName.includes('anomaly') || !keyName.includes('_avg');
```

Comment:
why test for this `anomaly` or `_avg`, vs `anomaly_avg`

Reply:
You are right. Missed it, we could safely change that to:
```suggestion
            return !keyName.includes('anomaly_avg');
```

Reply:
We check separately here for `anomaly` and `_avg`. Because we have a special occasion when we enable `NDVI anomaly (MODIS)` which is not following the pattern `anomaly_avg`